controllers/suggestions/suggest_offseason_event_controller.py

```diff
@@ -0,0 +1,29 @@
+from controllers.base_controller import LoggedInHandler
+from helpers.suggestions.suggestion_creator import SuggestionCreator
+from template_engine import jinja2_engine
+
+
+class SuggestOffseasonEventController(LoggedInHandler):
+    """
+    Allow users to suggest Offseason Events
+    """
+
+    def get(self):
```

Comment:
This should require login as well, I believe .Check the other suggestion controllers.